scadnano/scadnano.py

```diff
@@ -7807,6 +7881,11 @@ def __str__(self) -> str:
     def __repr__(self) -> str:
         return '_OxdnaVector({}, {}, {})'.format(self.x, self.y, self.z)
 
+    def __iter__(self) -> Iterator[float]:
```

Comment:
This is generally convenient, so that list(vector) gives a list [x,y,v].  I don't use it in the code, but do in the tests.